code_benches/run.py

```diff
@@ -8,14 +9,30 @@
 import sys
 import timeit
 
+parser = argparse.ArgumentParser(description="Run code_benches.")
+parser.add_argument("--target",
+    help="Target triple to use, defaults to host triple. You probably want to "
+        "set this if you're doing anything non-trivial.")
+parser.add_argument("--debug", action='store_true',
+    help="Use debug build of silverfish. Defaults to most recent that exists.")
+parser.add_argument("--release", action='store_true',
+    help="Use release build of silverfish. Defaults to most recent that exists.")
+parser.add_argument("--wasmception", action='store_true',
+    help="Use Wasmception for the WebAssembly libc. Defaults to most recent that exists.")
+parser.add_argument("--wasi-sdk", action='store_true',
+    help="Use WASI-SDK for the WebAssembly libc. Defaults to most recent that exists.")
+parser.add_argument("-o", "--output", default="benchmarks.csv",
+    help="Destination csv file to write benchmark results. Defaults to %(default)r.")
+args = parser.parse_args()
+
 # Note: This is a major configuration option, you probably want to set this if you're doing anything non-trivial
-SILVERFISH_TARGET = None
+SILVERFISH_TARGET = args.target
```

Comment:
This allows an empty arg, right?

Reply:
Yep, if omitted argparse pases `None` so it should be the same behavior as before. I'm currently using the default (`None`). I just updated it since it seemed like a used config option.